caf-audit-maven-plugin/src/main/java/com/hpe/caf/auditing/plugins/XMLToJavaMojo.java

```diff
@@ -50,24 +51,43 @@
     @Parameter(property = "packageName", defaultValue = "${project.groupId}.auditing")
     private String packageName;
 
+    /**
+     *  Indicates if the generated Java should be added as a test resource. If not set, or set to false, it is added as
+     *  a main resource.
+     */
+    @Parameter(property = "generateAsTestResource", defaultValue = "false")
```

Comment:
Asked Dermot and we agreed that this option doesn't need documented. It would be of no use to anyone other than ourselves.